src/main/java/com/faforever/client/fa/relay/ice/IceAdapterImpl.java

```diff
@@ -56,12 +55,14 @@
 
 @Component
 @Lazy
-@Slf4j
 public class IceAdapterImpl implements IceAdapter, InitializingBean, DisposableBean {
 
   private static final int CONNECTION_ATTEMPTS = 50;
   private static final int CONNECTION_ATTEMPT_DELAY_MILLIS = 100;
 
+  private static final Logger log = LoggerFactory.getLogger("faf-ice-adapter");
```

Comment:
is it not supposed to be IceAdapterImpl.class

Reply:
I agree, this is not the ice adapter log, it's the log of the client interacting with the adapter. I'd stay with `@slf4j`.

Reply:
In the original PR, we had two loggers: `log` with `IceAdapterImpl.class` name and `logger` with `faf-ice-adapter` name. I merged them and kept the string name, but i can switch to class name of course.

Reply:
See comment below for example of advanced log.